t/t2407-sdbus.t

```diff
@@ -140,7 +140,7 @@ test_expect_success 'create subscribe script' '
 	import sys
 	import flux
 	handle = flux.Flux()
-	fut = handle.rpc("sdbus.subscribe",{},flags=flux.constants.FLUX_RPC_STREAMING)
+	fut = handle.rpc("sdbus.subscribe",{"member":"PropertiesChanged","interface":"org.freedesktop.DBus.Properties","path":"/org/freedesktop/systemd1/unit/flux-t2406*"},flags=flux.constants.FLUX_RPC_STREAMING)
```

Comment:
perhaps can also test defaults (i.e. no filters) and see that you get extra signals?  Perhaps can just setup another script and launch some unit called "foobar" and make sure its in one signals.out and not the other?